pkg/service/regression/regression.go

```diff
@@ -103,7 +110,8 @@ func (r *Regression) GetAll(ctx context.Context, cid, appID string, offset *int,
 	tcs, err := r.tdb.GetAll(ctx, cid, appID, false, off, lim)
 
 	if err != nil {
-		r.log.Error("failed to get testcases from the DB", zap.String("cid", cid), zap.String("appID", appID), zap.Error(err))
+		sanitizedAppID := sanitiseInput(appID)
+		r.log.Error("failed to get testcases from the DB", zap.String("cid", cid), zap.String("appID", sanitizedAppID), zap.Error(err))
```

Comment:
Check failure on line 114 in pkg/service/regression/regression.go

**Log entries created from user input**

This log write receives unsanitized user input from 
https://github.com/keploy/keploy/blob/7206b0797938cc1119158a348da6bba8b0856708/http/regression/regression.go#L129-L129

Reply:
Resolved